### DISCUSSION TITLE
How to get column keys as Strings?

### DISCUSSION THREAD

**Discussion by wuarmin** [jeremyevans/sequel#1854] [General]
Hey,

I can't find the setting in Sequel to get column keys as a string. By default, they are symbols.

With TinyTds I can achieve it like this:
```ruby
RS2_DB.synchronize do |client|
  result = client.execute(sql)
  result.each(symbolize_keys: false) do |row|
  end
end
```
Is there a possibility to achieve this? Can I achive it by using https://sequel.jeremyevans.net/rdoc-plugins/files/lib/sequel/extensions/identifier_mangling_rb.html ? But how?

Thanks and best regards

**Comment by jeremyevans** [comment]
In Sequel, column keys are always returned as symbols, there is no setting to return strings, by design (all other parts of Sequel expect to retrieve data from the hashes using symbols).  You can use `transform_keys(&:to_s)` on the returned hashes to return a hash with string keys.

**Reply by wuarmin** [reply]
Alright thanks. It makes sense. Just messing around with legacy code where Sequel is used recently. I can't use `transform_keys` because I don't use activesupport, but I do it with plain ruby.

Thanks and best regards

**Reply by jeremyevans** [reply]
FYI, `Hash#transform_keys` was included in core Ruby starting in 2.5, which itself is already end of life.
